projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.spec.ts

```diff
@@ -31,6 +36,13 @@ class MockHamburgerMenuService {
   toggle(_forceCollapse?: boolean): void {}
 }
 
+// TODO: (CXSPA-5919) Remove mock next major release
+class MockFeatureConfigService {
+  isLevel() {
+    return true;
```

Comment:
You can change the return value in the test by adding the line `spyOn(featureConfigService, 'isLevel').and.returnValue(true);`.

Reply:
> As I recall, it was only new cases that failed.
It's actually the existing tests that are failing, the new ones are fine. It doesn't feel right to modify them just to accommodate some temporary code. I think leaving this condition is more elegant as this will future proof our tests and save some headaches. 

I can't imagine a scenario when additionally rendering DOM marked as deprecated will be the expected outcome. That's why the tests count twice as many nodes here. I'm not trying to argue this, I'm just curious about the reasoning here ;)